scripts/bootstrap.sh

```diff
@@ -13,9 +13,6 @@ BUILD_DIR="${ROOT_DIR}/build"
 cd "${BUILD_DIR}"
 rm -rf ./*
 
-# shellcheck disable=SC1090
-. "${SCRIPTS_DIR}/find-openssl.rc"
```

Comment:
I think this may or may not matter to @spencerjackson with his Atlas work right now.

Reply:
Good to know! You can still set `OPENSSL_ROOT_DIR` to override the toolchain openssl if that's the use case you had in mind.